applications/GeoMechanicsApplication/tests/test_compressibility.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+
+class KratosGeoMechanicsCompressibilityTests(KratosUnittest.TestCase):
+    """
+    This class contains simple single-element regression tests, checking the water pressure after a single time step.
+    It is a workflow test, which will fail if the additions of right hand side contributions (e.g. compressibility,
+    permeability, coupling) are changed.
+    """
+
+    def test_compressibility_upw_small_strain(self):
+        test_name = 'compressibility_tests/upw_small_strain'
+        expected_water_pressure_at_bottom = -49.1786
```

Comment:
I like that you have associated this value with a descriptive name. However, that same pattern is not used for the subsequent tests. Is there any particular reason why you have changed the approach?